config-provisioning/src/main/java/com/yahoo/config/provision/ApplicationId.java

```diff
@@ -12,24 +12,26 @@
  * @author vegard
  * @author bratseth
  */
-public final class ApplicationId implements Comparable<ApplicationId> {
+public class ApplicationId implements Comparable<ApplicationId> {
 
     private final TenantName tenant;
     private final ApplicationName application;
     private final InstanceName instance;
     private final String serializedForm;
 
-    public ApplicationId(ApplicationIdConfig config) {
```

Comment:
This API is used by old config models, so you need to be sure they are not using this. I think it is ok, but not sure it's worth the risk just to make final implicit?

Reply:
I could only find two usages of this: a unit test, and the `NginxMetricsReporter` component.